cfgov/v1/atomic_elements/organisms.py

```diff
@@ -743,6 +743,11 @@ class ChartBlock(blocks.StructBlock):
     description = blocks.CharBlock(
         required=True,
         help_text='Briefly summarize the chart for visually impaired users.')
+
+    last_updated_projected_data = blocks.DateBlock(
+        help_text="Month of latest entry in dataset"
```

Comment:
Nitpicky: can you please use single quotes for consistency with most of the other strings in this file?